### DISCUSSION TITLE
Issue with Power Toys lists - "Power rename" twice

### DISCUSSION THREAD

**Discussion by JonrGull** [moudey/Shell#211] [Q&A]
My apologies if this should be submitted to Power Toys instead, but I love using both of these tools. 

I installed SHELL today and noticed this duplicated selection of choosing "PowerRename" 

![powertoys](https://user-images.githubusercontent.com/25947147/226098957-4f7162c4-9b15-4f4a-a8de-498dcc149c0d.png)

This behavior only seems to appear when trying to PowerRename a folder.

It may have something to do with this setting.
![image](https://user-images.githubusercontent.com/25947147/226098921-c95c71cb-c0c8-489b-bf04-f56232c7861e.png)

I've tried playing with the Power Toys settings to see if I could fix it, but the only way to fix it is to turn the feature off.  If anyone has an idea of how I could solve this without a codebase update, I'd appreciate it. 

Thank you.

**Comment by moudey** [comment]
The last item can be removed by this method
open `imports/static.nss` and add this code
```
item(where=this.pos>10 find='"powerrename"' vis=vis.remove)
```
save changes, press `ctrl` + `right click` to reload settings.

I will add a new option to make it easier to control duplicate items.

**Comment by JonrGull** [comment]
Thank you! That helps a ton.

**Comment by moudey** [comment]
If this duplicate item is with folders only, the type property must be specified to narrow the search process
```
item(
    type='dir'	// dir only
    where=this.pos>10 and this.type==1	// Determine the item's position and type (sep=0, item=1, submenu=2)
    find='"powerrename"'	// Search the item title by matching
    vis=vis.remove 
)
```

**Comment by detile** [comment]
The duplicate entry when one or several files are selected can be removed with the code above in [this comment](https://github.com/moudey/Shell/discussions/211#discussioncomment-5353762). 
But when there is no file selected (right click on an empty space in explorer) the duplicate entry is still there. Not a big deal though.
